shell/deploy-to-dev.sh

```diff
@@ -11,10 +11,13 @@ APPNAME="$(get_app_name)"
 source "$SCRIPTS_DIR/lib/logging.sh"
 
 action=$1
-bento="bento1a"
-version="${DEPLOY_TO_DEV_VERSION:-"latest"}"
-namespace="$APPNAME--$bento"
-environment="${DEPLOY_TO_DEV_ENVIRONMENT:-"development"}"
+
+export DEVENV_DEPLOY_BENTO="bento1a"
+export DEVENV_DEPLOY_VERSION="${DEPLOY_TO_DEV_VERSION:-"latest"}"
+export DEVENV_DEPLOY_NAMESPACE="$APPNAME--$DEVENV_DEPLOY_BENTO"
+export DEVENV_DEPLOY_ENVIRONMENT="${DEPLOY_TO_DEV_ENVIRONMENT:-"development"}"
+export DEVENV_DEPLOY_DEV_EMAIL="${DEV_EMAIL:-$(git config user.email)}"
```

Comment:
I think this should be `git config --global user.email`

Reply:
won't it roll that into local on get?

Reply:
yeah, doesn't matter, if current repo has override, it should be used, if it doesn't this will return the global option

Reply:
Do we want to have the deploy-app functionality change based on the current repo?? I don't think so, also we're going to nuke this method of setting email in the future soon. If we enable local config override that'll make it a breaking change later

Reply:
it's always been this way. I just copied the default. And yeah, you want to do local repo settings if available - think personal vs work repos (shouldn't be very common, but it's a valid scenario)

Reply:
Well, then we're going to break that later. The idea of dev email is not to be "correct" it's to be a machine wide identifier for metrics/logging tagging, so having it change per repo is not useful.

Reply:
There are so many ways that could go wrong ... But for the purpose of this PR this is what we want.